### ISSUE TITLE
Possible to disable HTML5 video?

### ISSUE THREAD

**Issue by gusseleet** [gree/unity-webview#114]
As the topic says, is it possible disable HTML5 video the user can't see any videos?

**Comment by KojiNakamaru** [comment]
It could be possible by modifying the following parts to ignore `*.mp4`, etc:

https://github.com/gree/unity-webview/blob/92170dbf1b66a587d871cb2fb42fee40e5289663/plugins/Android/src/net/gree/unitywebview/CWebViewPlugin.java#L127
https://github.com/gree/unity-webview/blob/92170dbf1b66a587d871cb2fb42fee40e5289663/plugins/iOS/WebView.mm#L170
https://github.com/gree/unity-webview/blob/92170dbf1b66a587d871cb2fb42fee40e5289663/plugins/iOS/WebView.mm#L183

or you may search and remove video tags by JavaScript (through `WebViewObject#EvaluateJS()`.

**Comment by gusseleet** [comment]
Hi! Thank you so much for your answer.

This is a stupid question. But can I just change the .java file and then re-import the package?

**Comment by KojiNakamaru** [comment]
Yes, the page you described seems to internally load m4a/m4v data by javascript. If you modify the plugin to ignore URLs with m4a/m4v, the video won't be played.

**Comment by gusseleet** [comment]
Thanks for the answer. One stupid question. 

Can I just change the java file that is located under src/net/gree/unitywebview and then re-import the package?

**Comment by KojiNakamaru** [comment]
Yes, and though it is only for Android, I also made an experimental implementation to skip mp4/m4a/m4v/mov: https://github.com/gree/unity-webview/tree/experimental/disable_movies . Could you please try it.

BTW `shouldOverrideUrlLoading` originally referred above wasn't enough and I utilized `shouldInterceptRequest` instead: https://github.com/gree/unity-webview/commit/1386666732cf95b7cad7c2a44671ff4881793783 .
